packages/domain-picker/src/domain-picker/style.scss

```diff
@@ -209,10 +177,61 @@ $domain-picker__suggestion-item-height: 24px;
 	}
 }
 
-.domain-picker__price-is-paid {
+.domain-picker__price-text {
+	display: none;
+
+	@include break-small {
+		display: inline;
+	}
+}
+
+.domain-picker__price-cost {
 	text-decoration: line-through;
 }
 
+.domain-picker__suggestion-item-select-button {
+	display: flex;
+	align-items: center;
+	height: 36px;
+
+	span {
+		display: none;
+	}
+
+	svg {
+		fill: var( --studio-blue-50 );
+		margin-left: 10px;
+		margin-right: -2px;
+	}
+
+	@include break-small {
+		// We're mocking the look of a button,
+		// this is not an actual button.
+		background: #007cba;
+		color: var( --studio-white );
+		border-radius: 2px;
+		padding: 0 12px;
+		opacity: 0;
+		transition: opacity 200ms ease-in-out;
+
+		// Overlaps price for transition purpose
+		position: absolute;
+		right: 14px;
+
+		&:hover {
+			background: #0070a7;
```

Comment:
Let's pull this and `background` from a CSS variable.

Reply:
This is a little tricky to track down in wordpress core and I don't really have all the answers but it uses vars now. https://github.com/Automattic/wp-calypso/pull/43346/commits/40caccb698f4d6d5ec86d4cc265d0f6a7f22ccc4